pytracking/features/preprocessing.py

```diff
@@ -106,8 +106,8 @@ def sample_patch(im: torch.Tensor, pos: torch.Tensor, sample_sz: torch.Tensor, o
     szl = torch.max(sz.round(), torch.Tensor([2])).long()
 
     # Extract top and bottom coordinates
-    tl = posl - (szl - 1) / 2
-    br = posl + szl/2 + 1
+    tl = posl - (szl - 1) // 2
```

Comment:
this is because of python3 version i think which change between 3.7 and 3.6(which is that we use)
but i am not 100 sure
the code crash when he reach this line